server/log/main.go

```diff
@@ -89,6 +119,14 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Load up our private key, exit if this fails to work
+	keyManager, err := loadPrivateKey(*privateKeyFile, *privateKeyPassword)
+
+	if err != nil {
+		glog.Errorf("Failed to load server key: %v", err)
+		os.Exit(1)
```

Comment:
add a TODO to remove this during the multi-tenant work?
Also, is there not a `glog.Fatalf()` ?